src/Microsoft.AspNetCore.SignalR/SignalRDependencyInjectionExtensions.cs

```diff
@@ -16,8 +16,6 @@ public static ISignalRBuilder AddSignalR(this IServiceCollection services)
             services.AddSingleton(typeof(IHubContext<>), typeof(HubContext<>));
             services.AddSingleton(typeof(HubEndPoint<>), typeof(HubEndPoint<>));
             services.AddSingleton<IConfigureOptions<SignalROptions>, SignalROptionsSetup>();
-            services.AddSingleton<JsonNetInvocationAdapter>();
```

Comment:
Again, for what purpose? Are we changing our plan on user-replaceable hub protocols?